### PR TITLE
feat: add download count

### LINKED ISSUE #17: Add download counts for each plugin
Should be easy to do with Deno KV.

### PR DESCRIPTION
**Pull request by g-plane**
Close #17 

This PR doesn't use Deno KV.

Advantages:

- Download count is more accurate.
- No need to pay for storage usage.

### CODE REVIEW

**Review by dsherret — COMMENTED** [review]
This is awesome, but it looks like we're making double the amount of requests to github now. Maybe we could consolidate everything and only make a request to github's API for `releases` instead of also `releases/latest`?

```
Making request to https://api.github.com/repos/dprint/dprint-plugin-typescript/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-typescript/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-json/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-json/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-markdown/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-markdown/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-toml/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-toml/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-dockerfile/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-dockerfile/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-biome/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-biome/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-ruff/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-ruff/releases
Making request to https://api.github.com/repos/dprint/dprint-plugin-jupyter/releases/latest
Making request to https://api.github.com/repos/dprint/dprint-plugin-jupyter/releases
Making request to https://api.github.com/repos/g-plane/dprint-plugin-malva
Making request to https://api.github.com/repos/g-plane/malva/releases/latest
Making request to https://api.github.com/repos/g-plane/malva/releases
Making request to https://api.github.com/repos/g-plane/dprint-plugin-markup_fmt
Making request to https://api.github.com/repos/g-plane/markup_fmt/releases/latest
Making request to https://api.github.com/repos/g-plane/markup_fmt/releases
Making request to https://api.github.com/repos/g-plane/dprint-plugin-pretty_yaml
Making request to https://api.github.com/repos/g-plane/pretty_yaml/releases/latest
Making request to https://api.github.com/repos/g-plane/pretty_yaml/releases
```

File: utils/github.ts [hunk 1]
```diff
@@ -97,6 +99,40 @@ function getReleaseInfo(data: { tag_name: string; body: string; assets: { name:
   }
 }
 
+function getDownloadCount(assets: ReleaseAsset[]) {
+  return assets.find(({ name }) => name === "plugin.wasm" || name === "plugin.json")?.download_count ?? 0;
```
Comment (dsherret) [inline utils/github.ts:90]: Great find!

**Comment by g-plane** [comment]
Updated.

**Review by dsherret — APPROVED** [review]
Thanks so much!

**Pull request merged**